backend/build.gradle

```diff
@@ -49,6 +49,7 @@ dependencies {
     compile "org.springframework.boot:spring-boot-starter-security:$spring_boot_version"
     compile "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
     compile "org.springframework.boot:spring-boot-starter-data-jpa:$spring_boot_version"
+    compile "org.postgresql:postgresql:42.0.0"
```

Comment:
So we always use a database now? Does the local execution still work?

Reply:
Yes, it still makes one in memory when running of localhost

Reply:
Okay, interesting :)